test/known_issues/test-http-clientrequest-end-contentlength.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+
```

Comment:
For new known-issue tests, there should a comment pointing to an open issue discussing the issue and it's very helpful to add a comment explaining why the test should be expected to fail.